src/Orleans.Transactions/DistributedTM/TransactionOverloadDetector.cs

```diff
@@ -66,8 +64,7 @@ public bool IsOverloaded()
 
             DateTime now = DateTime.UtcNow;
             this.monitor.TryAction(now);
-            long startCounter = this.statistics.TransactionStartedCounter;
-            double txPerSecondCurrently = CalculateTps(this.transactionStartedAtLastCheck, this.lastCheckTime, startCounter, now);
+            double txPerSecondCurrently = CalculateTps(this.lastStatistics.TransactionsStarted, this.lastCheckTime, this.statistics.TransactionsStarted, now);
```

Comment:
where do you track ITransactionAgentStatistic.Throttled ? did I miss anything. I thought it will be tracked here

Reply:
It's tracked in the transaction agent, where the IsOverloaded is called.  The agent is responsible for updating it's statistics.